nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-2-6-processors/src/main/java/org/apache/nifi/processors/kafka/pubsub/PublishKafkaRecord_2_6.java

```diff
@@ -239,6 +251,7 @@ public class PublishKafkaRecord_2_6 extends AbstractProcessor implements Verifia
             + "If not specified, no FlowFile attributes will be added as headers.")
         .addValidator(StandardValidators.REGULAR_EXPRESSION_VALIDATOR)
         .expressionLanguageSupported(NONE)
+        .dependsOn(PUBLISH_STRATEGY, PUBLISH_USE_VALUE)
```

Comment:
If this property now depends on Publish Strategy, it should come after Publish Strategy in the Properties list (i.e., in `this.properties`)

Reply:
Do you think a PublishStrategy enum would be useful here (like OutputStrategy suggested above)?

Reply:
Oof.  That's an oversight.  The propagation from processor to pool to lease is needed, so I'll do enum here as well.